parts/linux/cloud-init/artifacts/mariner/cse_install_mariner.sh

```diff
@@ -128,6 +127,10 @@ installStandaloneContainerd() {
 
 }
 
+ensureRunc() {
+  echo "Mariner Runc is included in the Mariner base image. Skipping downloading and installing Runc"
```

Comment:
Do we have to have this function because cse_instsall_ubuntu has its own ensureRunc()?

Reply:
In my install-depenedencies.sh around ln 232, if the package is 'runc', ensureRunc will be called. Therefore in both cse_install_ubuntu and cse_install_mariner it needs to have the function. Another way to do that is add an if before that line, if it's Ubuntu, call ensureRunc. Otherwise do nothing (maybe leave the same echo there).

Reply:
I'll leave it up to you on what you think is more readable since you're more familiar with this block of code :)